content/en/docs/guides/compression.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Compression
+description: >-
+    How to use compression to reduce the bandwidth used by gRPC
+---
+
+### Overview
+
+Compression is used to reduce the amount of bandwidth used between peers. The compression supported by gRPC acts at the individual message level.
```

Comment:
I think you don't have full control at individual message level - you can just choose to disable it at message level (IIUC)